### ISSUE TITLE
EntryPointNotFoundException on rider / .net 462 / macOS

### ISSUE THREAD

**Issue by serialseb** [shouldly/shouldly#437]
Quick one which should be easy to repro. Using 2.8.3, rider, xunit, .net 4.6.2 classlib, running tests in the runner throws with an EntryPointNotFoundException inside a TypeInitializerException that leads me to
```
      [DllImport("kernel32.dll", SetLastError = true)]
      private static extern IntPtr CreateToolhelp32Snapshot(uint dwFlags, uint th32ProcessId);
```
Which I think means it's trying to do something funky with diff windows in VS or something.

Downgrading to 2.8.2 solves the issue.

**Comment by JosephWoodward** [comment]
Thanks for reporting this Seb, I'll take a look at it.

**Comment by JosephWoodward** [comment]
This fix has now been merged and will be in the next release :thumb:

**Comment by JosephWoodward** [comment]
This has now been released in [3.0.0-beta0003](https://www.nuget.org/packages/Shouldly/3.0.0-beta0003)
